deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Product.wxs

```diff
@@ -11,7 +11,8 @@
     <Package InstallerVersion="200"
              Compressed="yes"
              InstallScope="perMachine"
-             Description="$(var.ProductName)"/>
+             Description="$(var.ProductName)"
+             Platform="$(var.Platform)" />
```

Comment:
The [documentation](http://wixtoolset.org/documentation/manual/v3/xsd/wix/package.html) recommends not using `Platform` here:
> Use of this attribute is discouraged; instead, specify the -arch switch at the candle.exe command line or the InstallerPlatform property in a .wixproj MSBuild project.

Can you add it as an argument in the PowerSheel script instead?